internal/handlers/common/msg_listcommands.go

```diff
@@ -170,6 +170,10 @@ var Commands = map[string]command{
 		Help:    "Returns a pong response.",
 		Handler: handlers.Interface.MsgPing,
 	},
+	"saslStart": {
+		Help:    "Starts a SASL conversation.",
+		Handler: (handlers.Interface).MsgSASLStart,
```

Comment:
```suggestion
		Handler: handlers.Interface.MsgSASLStart,
```

I see the other Msg don't use `(` and `)` anymore, so I guess this one shouldn't use them either

Reply:
Fixed